nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestClidObject.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour Al Kotob
+ */
+
+package org.nuxeo.rest.management;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.connect.identity.LogicalInstanceIdentifier;
+import org.nuxeo.connect.identity.LogicalInstanceIdentifier.InvalidCLID;
+import org.nuxeo.connect.identity.LogicalInstanceIdentifier.NoCLID;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.jaxrs.test.JerseyClientHelper;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.ServletContainerFeature;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+
+/**
+ * @since 11.1
+ */
+@RunWith(FeaturesRunner.class)
+@Features(ManagementFeature.class)
+public class TestClidObject {
+
+    public static final String CLID_PATH = "site/management/clid";
+
+    @Inject
+    protected ServletContainerFeature servletContainerFeature;
+
+    protected Client client;
+
+    protected ObjectMapper mapper;
+
+    @Before
+    public void setup() {
+        mapper = new ObjectMapper();
+        client = JerseyClientHelper.clientBuilder().setCredentials("Administrator", "Administrator").build();
+    }
+
+    @Test
+    public void testClid() throws IOException, NoCLID, InvalidCLID {
```

Comment:
Please remove `NoCLID`, Sonar should warn about it.

Reply:
for the use of `mapper` and `client` i think the code can be refactored with BaseTest class to hold these 2 fields. the classe `TestDistributionObject` already use them with the same values.

Reply:
Nice idea but:
- `BaseTest` is not exposed since the `nuxeo-rest-api-test` module is not built as a test-jar (could do it but not sure we want to for 10.10 and would add a dependency).
- Not sur if we want to reuse it anyway as it's old and more "document/Nuxeo REST API" oriented.
- In the future, authentication might change: using a JWT token, so building our own client is fine.

If we want to refactor I'm in favor of:
- See if we can use `HttpClientTestRule`, accessible via `nuxeo-features-test` and lighter.
- Otherwise have an abstract test class similar as `BaseTest` but with the strict minimum.

Reply:
for `BaseTest` i was thinking about a new one, yep i see you point of view. try the two propositions ( `HttpClientTestRule ` and a t least make a new class with the strict minimum it's a good alternative) 
thx

Reply:
will do in a separate commit then